src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -346,11 +365,26 @@ export default function ConfirmSwapModal({
   }, [onCancel, onDismiss, priceUpdate, showAcceptChanges, trade])
 
   const modalHeader = useCallback(() => {
-    if (confirmModalState !== ConfirmModalState.REVIEWING && !showAcceptChanges) {
+    // Header remains visible above V2 of the progress indicator until swap is finalized (i.e. success or failure)
+    const lastStepComplete = swapConfirmed || errorType
+    const isSingleStep = confirmModalState !== ConfirmModalState.REVIEWING && pendingModalSteps.length === 1
+    if (showProgressIndicatorV2 && (isSingleStep || lastStepComplete)) {
+      return null
+    } else if (!showProgressIndicatorV2 && confirmModalState !== ConfirmModalState.REVIEWING && !showAcceptChanges) {
       return null
     }
     return <SwapModalHeader inputCurrency={inputCurrency} trade={trade} allowedSlippage={allowedSlippage} />
```

Comment:
header X is missing clickable style
![Screenshot 2023-11-01 at 10 18 12 AM](https://github.com/Uniswap/interface/assets/11512321/02117dff-8453-469f-9e80-57ce63720953)

Reply:
Added `ClickableStyle` to the shared `CloseIcon` component